quickstart/examples/inference-scheduling/README.md

```diff
@@ -13,56 +13,91 @@ This profile defaults to the approximate prefix cache aware scorer, which only o
 1. Install the dependencies; see [install-deps.sh](../../install-deps.sh)
 2. Use the quickstart to deploy Gateway CRDs + Gateway provider + Infra chart. This example uses `kgateway` but should work with `istio` given some modifications as described below step 3.
 
+```bash
+# Set common environment variables
+export NAMESPACE=llm-d-inference-scheduling
+export GATEWAY=kgateway # choose between [gke-l7-regional-external-managed, istio, kgateway]
+```
+
 ```bash
 # From the repo root
 cd quickstart
-export HF_TOKEN=${HFTOKEN}
-./llmd-infra-installer.sh --namespace llm-d-inference-scheduling -r infra-inference-scheduling --gateway kgateway --disable-metrics-collection
+export HF_TOKEN=$(YOUR_TOKEN)
+./llmd-infra-installer.sh --namespace ${NAMESPACE} -r infra-inference-scheduling --gateway ${GATEWAY} --disable-metrics-collection
 ```
     - It should be noted release name `infra-inference-scheduling` is important here, because it matches up with pre-built values files used in this example.
 
 3. Use the helmfile to apply the modelservice and GIE charts on top of it.
 
-```bash
-cd examples/inference-scheduling
-helmfile --selector managedBy=helmfile apply helmfile.yaml --skip-diff-on-install
-```
+* For `kgateway` and `istio`:
+
+  ```bash
+  cd examples/inference-scheduling
+  helmfile --namespace ${NAMESPACE} --selector managedBy=helmfile apply helmfile.yaml --skip-diff-on-install
+  ```
+* For `gke-l7-regional-external-managed`, set the `provider.name=gke` for the gaie chart, i.e., 
+
+  ```bash
+  cd examples/inference-scheduling
+  helmfile --namespace ${NAMESPACE} --selector managedBy=helmfile \
+  --set provider.name=${GAIE_GATEWAY_PROVIDER} \
```

Comment:
GAIE_GATEWAY_PROVIDER is not defined; but since this is a gke section, just put gke here.

Reply:
fixed in the new gke.md